Oracle/med_admin_init.sql

```diff
@@ -0,0 +1,2 @@
+/*Verify that there is useful data in the observation_fact table.  If not, Heron ETL may have failed.*/
+SELECT count(patient_num) from BLUEHERONDATA.observation_fact where modifier_cd = 'MedObs|MAR:Given' and rownum = 1;
```

Comment:
I'm not sure I'm using this init step as intended by Luigi.  When observation_fact is empty, Luigi returns the following -

Scheduled 2 tasks of which:
* 1 ran successfully:
    - 1 MedAdminInit(account=oracle)
* 1 failed:
    - 1 MedAdmin(account=oracle)

This progress looks :( because there were failed tasks

The MedAdmin job fails as intended but the cause is ambiguous, as the MedAdminInit job ran successfully.